modin/core/storage_formats/base/query_compiler.py

```diff
@@ -4588,6 +4588,18 @@ def rolling_sum(self, fold_axis, rolling_args, *args, **kwargs):
             self, rolling_args, *args, **kwargs
         )
 
+    @doc_utils.doc_window_method(
```

Comment:
FYI, there's no need to touch this file for what we are doing.

Reply:
Ok right, because changing client QC should suffice. I think I added it for completeness if this is extended again into another QC.